.travis.yml

```diff
@@ -28,8 +28,10 @@ install:
 before_script:
   - npm run check-package
   - npm run lint
-  - docker build . -t dapi-dev 
+  - docker build . -t dapi-dev
   - echo SERVICE_IMAGE_DAPI=dapi-dev >> .env
```

Comment:
This is temporary?

Reply:
I just removed space at the end of the line. It was done automatically by the IDE.